spec/Subject-spec.js

```diff
@@ -221,9 +215,6 @@ describe('Subject', function () {
     subscription1.unsubscribe();
 
     subject.complete();
-    subject.next(9);
-    subject.complete();
-    subject.error(new Error('err'));
```

Comment:
Oh, thanks for catching it. Subject's spec's are not updated to include those. Updating it now.